OIIInspector/OIIIClient.py

```diff
@@ -1,9 +1,10 @@
-import json
+import os
 
 
 class OIIIClient:
 
-    def get_index_image_package_list(self, image_address, address_type):
-
-        print(image_address, address_type)
-        return json.loads("{}")
+    def get_index_image_package_list(self, image_address):
+        grp_curl_position = "./grpCurl/grpcurl"
```

Comment:
I think we should just assume that the binary will be callable as "grpcurl" in the environment.

Reply:
I implemented it as import module which can provide address of grpcurl binary, Is that a right way how to do this?

Reply:
Now it should be okay, is "grpcurl" as constant okay?